### PR TITLE
CTECH-1723: Updates notebooks to work for forked PR's.

### PR DESCRIPTION
**Pull request by pms1969**
Signed-off-by: Paul Saunders <paul.saunders@finbourne.com>

# Pull Request Checklist

- [x] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [x] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

Changes focused around enabling access tokens for PR's.  May well need to be merged before the efficacy of the change can be fully realised.

### CODE REVIEW

File: examples/use-cases/data-quality-control/Custom Entities.ipynb [hunk 1]
```diff
@@ -63,10 +63,6 @@
     "# Set the secrets path\n",
     "secrets_path = os.getenv(\"FBN_SECRETS_PATH\")\n",
     "\n",
-    "# For running the notebook locally\n",
-    "if secrets_path is None:\n",
```
Comment (rizwansaeed) [inline examples/use-cases/data-quality-control/Custom Entities.ipynb:67]: is there are reason we need to remove this? it is useful for when people are running them locally (so they dont have to add and then remove code)?

**Review by rizwansaeed — CHANGES_REQUESTED** [review]

File: examples/use-cases/data-quality-control/Custom Entities.ipynb (hunk 1 above)
Comment (pms1969) [reply examples/use-cases/data-quality-control/Custom Entities.ipynb:67]: Yes, it positively breaks things as I recall.  If people want to run them locally, then they can just add the FBN_SECRETS_PATH env var, no code necessary.

Comment (pms1969) [reply examples/use-cases/data-quality-control/Custom Entities.ipynb:67]: It was also applied inconsistently.  This snippet of code appeared in about 7 workbooks out of however many we had.  Those workbooks were broken with access tokens.

Comment (rizwansaeed) [reply examples/use-cases/data-quality-control/Custom Entities.ipynb:67]: some editors (I think PyCharm is one) dont currently let you pass env vars through to the Jupyter runner which means you need to edit the notebook locally.

What were the issue you were seeing?

**Comment by pms1969** [comment]
Superceded by https://github.com/finbourne/sample-notebooks/pull/518 which doesn't touch the notebooks.

**Pull request closed without merging**